### ISSUE TITLE
Right icon present to show selected tag that icon color contrast with back ground color less than 3:1 :A11y_.NET maui_Projact_Non text Contrast

### ISSUE THREAD

**Issue by kapilvaishna** [dotnet/maui#30868]
**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## GitHub Tags
#A11yTCS; #A11yMAS; #WCAG1.4.11; #Non-text Contrast; #BM_.NETMaui_Win32_JUL2025; #DesktopApp; #Win32; #FTP; #A11ySev2; #.NET MAUI SDKs; #A11yWCAG2.1;

## Environment Details:
Application: Developer Balance (.NET Maui)
Windows 11
OS Build: 26100.2161
Version 24H2

## Repro Steps:

1. Install and open the Developer Balance app
2. TAB till any project  and press ENTER key.
3. Apply light  theme
4. TAB till selected tag
5. Observe the issue 

## Actual Result:
Right icon present to show selected tag that icon color contrast with background color is 1.193:1, which is less than 3:1. and only color used to differentiate from non selected tag.

## Expected Result:
Right icon present to show selected tag that icon color contrast with back ground color should be equal and greater than 3:1 so color blind person also don't face difficulty to differentiate non selected and selected tags. 

## User Impact:
Color blind person will face difficulty to differentiate non selected and selected tags. 

## Attachment

<img width="1279" height="714" alt="Image" src="https://github.com/user-attachments/assets/d5e9721c-f27a-4d5c-ada9-6ab2e2c19301" />

**Comment by ghost** [comment]
This issue has been verified in Visual Studio 17.14.0 Preview 4.0 with MAUI versions 9.0.90. The selection indicator (tick symbol) is not present in the tag elements with the latest [developer sample](https://github.com/dotnet/maui-samples/tree/main/9.0/Apps/DeveloperBalance). I have attached the screenshot for reference.

<img width="593" height="300" alt="Image" src="https://github.com/user-attachments/assets/0af01b45-76c4-445a-be9f-073ae269dc59" />

**Comment by kapilvaishna** [comment]
Issue is repro in below version 

<img width="256" height="97" alt="Image" src="https://github.com/user-attachments/assets/df08ccb5-6e25-4899-9d01-9df0d8d3b48f" />

<img width="1279" height="716" alt="Image" src="https://github.com/user-attachments/assets/bdcb8f6d-3c1a-486b-a4d2-b277d55e63b4" />

**Comment by Vignesh-SF3580** [comment]
> Issue is repro in below version
> 
> <img alt="Image" width="256" height="97" src="https://private-user-images.githubusercontent.com/124254981/502454189-df08ccb5-6e25-4899-9d01-9df0d8d3b48f.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjEyMTk4MDcsIm5iZiI6MTc2MTIxOTUwNywicGF0aCI6Ii8xMjQyNTQ5ODEvNTAyNDU0MTg5LWRmMDhjY2I1LTZlMjUtNDg5OS05ZDAxLTlkZjBkOGQzYjQ4Zi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMDIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTAyM1QxMTM4MjdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zMzFjMmNiNWEwNjZhMjU3YzdjZDZhNTMxZGE5Nzc0ODYxN2JmOGQxN2MxZTRjNDVlNzNjNzE3NTExZDI2NzMxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.PY3mANWlr-5luMSqiALv0uCQJB_Zlt2fqXLXdCWkgZE"> <img alt="Image" width="1279" height="716" src="https://private-user-images.githubusercontent.com/124254981/502454308-bdcb8f6d-3c1a-486b-a4d2-b277d55e63b4.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjEyMTk4MDcsIm5iZiI6MTc2MTIxOTUwNywicGF0aCI6Ii8xMjQyNTQ5ODEvNTAyNDU0MzA4LWJkY2I4ZjZkLTNjMWEtNDg2Yi1hNGQyLWIyNzdkNTVlNjNiNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMDIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTAyM1QxMTM4MjdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yMmZhMTQyYTcxMDY4Yzk2ZjllZGQ5OWVhOGVkMjAyMmYxMGI0YzM2MDEyZmUyMTUyYWFkYjgyNTE4NGZjZmQ5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.gSaErfRnaXTfrm4uQmJZrNtAGNOqi0enx_koWvJWWDM">

@kapilvaishna @PureWeen Based on the attached image, the check mark is still not displayed correctly. It seems this fix was not included in the version you tested.
 
I verified the behavior using the .NET 9 DeveloperBalance sample app with the NuGet package generated from the [PR](https://github.com/dotnet/maui/pull/31126) , and it works as expected. Attached a video for reference.

https://github.com/user-attachments/assets/f8378cc7-0e36-4f4c-93f6-8f6b6e359181

**Comment by AnKushSingh05** [comment]
#Closed;#A11yFixed;
Issue is fixed

<img width="1366" height="768" alt="Image" src="https://github.com/user-attachments/assets/7db19605-9a75-4c6c-b249-b6060789f4f8" />
